full/src/main/java/apoc/uuid/UuidHandler.java

```diff
@@ -91,14 +92,16 @@ private void checkAndRestoreUuidProperty(Iterable<PropertyEntry<Node>> nodePrope
     @Override
     public Void beforeCommit(TransactionData txData, Transaction transaction, GraphDatabaseService databaseService) throws Exception {
 
-        Iterable<Node> createdNodes = txData.createdNodes();
+        // assignedLabels handles both created nodes and set labels of existing nodes
+        Iterable<LabelEntry> assignedLabels = txData.assignedLabels();
```

Comment:
since setting UUID on existing nodes with new labels is a breaking change, WDYT of making this behavior configurable (and disabled by default)?